### PR TITLE
Allow to specify working directory from cli

### PR DESCRIPTION
**Pull request by ernestas-poskus**
@laurynas 

Adding cli option which allow specifying working_directory.

Added cli option argument validation, should I add same for config file?

### CODE REVIEW

File: lib/sidekiq/pool/cli.rb [hunk 1]
```diff
@@ -111,6 +112,14 @@ def parse_options(argv)
             @pool_config = arg
           end
 
+          o.on '-m', '--working-direcory PATH', "path to working direcory" do |arg|
```
Comment (laurynas) [inline lib/sidekiq/pool/cli.rb]: Typo in `direcory`

Not sure about `-m`. Other good options like `-d` are taken. Maybe just have long version?

Comment (ernestas-poskus) [inline lib/sidekiq/pool/cli.rb]: good catch, sure we can keep longer version

**Comment by ernestas-poskus** [comment]
> but currently the problem is that cli options are parsed only when forking child (too late)

We do parse cli before calling run https://github.com/laurynas/sidekiq-pool/blob/master/bin/sidekiq-pool#L11

Tested

```bash
λ features/allow_to_specify_working_directory_from_cli ~/vinted/sidekiq-pool
$ ./bin/sidekiq-pool --working-directory /t
Provided directory /t does not exist
λ features/allow_to_specify_working_directory_from_cli ~/vinted/sidekiq-pool
$ ./bin/sidekiq-pool --working-directory /tmp
2017-01-06T19:59:55.731Z 26499 TID-gtk1facjs INFO: ==================================================================
2017-01-06T19:59:55.731Z 26499 TID-gtk1facjs INFO:   Please point sidekiq to a Rails 3/4 application or a Ruby file  
2017-01-06T19:59:55.731Z 26499 TID-gtk1facjs INFO:   to load your worker classes with -r [DIR|FILE].
2017-01-06T19:59:55.731Z 26499 TID-gtk1facjs INFO: ==================================================================
2017-01-06T19:59:55.731Z 26499 TID-gtk1facjs INFO: sidekiq-pool [options]
    -c, --concurrency INT            processor threads to use
    -d, --daemon                     Daemonize process
    -e, --environment ENV            Application environment
    -g, --tag TAG                    Process tag for procline
    -q, --queue QUEUE[,WEIGHT]       Queues to process with optional weights
    -r, --require [PATH|DIR]         Location of Rails application with workers or file to require
    -t, --timeout NUM                Shutdown timeout
    -v, --verbose                    Print more verbose output
    -w, --fork-wait NUM              seconds to wait between child forks, default 1
    -C, --config PATH                path to YAML config file
    -L, --logfile PATH               path to writable logfile
    -P, --pidfile PATH               path to pidfile
    -p, --pool-config PATH           path to pool config file
        --working-directory PATH     path to working directory
    -V, --version                    Print version and exit
    -h, --help                       Show help

```

**Comment by laurynas** [comment]
Oh, missed that. 👍

**Pull request merged**